app/src/renderer/features/session/sessionSlice.ts

```diff
@@ -0,0 +1,50 @@
+import { createSlice } from "@reduxjs/toolkit";
+import type { PayloadAction } from "@reduxjs/toolkit";
+import type { RootState } from "../../store";
+
+enum SessionStatus {
+  Unauth,
+  Offline,
+  Auth,
+}
+
+interface AuthData {
+  expiration: string;
+  token: string;
+  journalistUUID: string;
```

Comment:
Nit: This could be another opportunity to take advantage of TypeScript's nominal typing, like in <https://github.com/freedomofpress/securedrop-client/pull/2516#discussion_r2201689633>.  The stakes are even lower in this case, since we don't depend on the UUID validity of this server-provided value; just highlighting it while we're laying down our TypeScript style and conventions.

Reply:
I've opened a new issue for it https://github.com/freedomofpress/securedrop-client/issues/2553